packages/automated-dispute/src/types/prophet.ts

```diff
@@ -3,12 +3,14 @@ import { NormalizedAddress } from "@ebo-agent/shared";
 import { Address } from "viem";
 
 export type RequestId = NormalizedAddress;
+export type RequestStatus = "active" | "finalized";
```

Comment:
```suggestion
export type RequestStatus = "Active" | "Finalized";
```

should we do it this way to have consistent naming convention for statuses?

Reply:
Seems reasonable!